crypto/ts/ts_err.c

```diff
@@ -39,7 +39,7 @@ static const ERR_STRING_DATA TS_str_reasons[] = {
     {ERR_PACK(ERR_LIB_TS, 0, TS_R_NONCE_MISMATCH), "nonce mismatch"},
     {ERR_PACK(ERR_LIB_TS, 0, TS_R_NONCE_NOT_RETURNED), "nonce not returned"},
     {ERR_PACK(ERR_LIB_TS, 0, TS_R_NO_CONTENT), "no content"},
-    {ERR_PACK(ERR_LIB_TS, 0, TS_R_NO_TIME_STAMP_TOKEN), "no time stamp token"},
+    {ERR_PACK(ERR_LIB_TS, 0, TS_R_NO_TIME_STAMP_TOKEN), "no timestamp token"},
```

Comment:
The various TIME_STAMP macros should be updated in due course.  It doesn't have to be in this PR.